oauthproxy.go

```diff
@@ -742,9 +742,37 @@ func (p *OAuthProxy) SignOut(rw http.ResponseWriter, req *http.Request) {
 		p.ErrorPage(rw, req, http.StatusInternalServerError, err.Error())
 		return
 	}
+
+	p.backendLogout(rw, req)
+
 	http.Redirect(rw, req, redirect, http.StatusFound)
 }
 
+func (p *OAuthProxy) backendLogout(rw http.ResponseWriter, req *http.Request) {
+	session, err := p.getAuthenticatedSession(rw, req)
+	if err != nil {
+		logger.Errorf("error getting authenticated session during backend logout: %v", err)
+		return
+	}
+	if session == nil {
+		return
+	}
+	providerDatas := p.provider.Data()
+	if providerDatas.BackendLogoutURL == "" {
+		return
+	}
+	backendLogoutURL := strings.ReplaceAll(providerDatas.BackendLogoutURL, "{id_token}", session.IDToken)
+	resp, err := http.Get(backendLogoutURL) // #nosec G107
```

Comment:
is the `nosec G107` really necessary? can we somehow avoid this?

Reply:
I don't think it's doable, in anycase, it would be a variable but not a end user input  that can be tampered with (more an admin input)

Reply:
Okay leave it as is for now :)

Reply:
I would appreciate a code comment explaining why we are having to use the gosec so that we can remember this context in the future